tests/test_image_routes.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
We may want to organize the route tests into a views folder or something if we're planning to have multiple test files for the views module.

Reply:
lmk if we're wanting to do a directory for the views or if the image tests should go back in the file, I moved them out because the file was going to continue to get longer and longer, but if this isn't an issue they can just go back in

Reply:
I agree that the views tests are going to grow kind of large and since most of our code will live there it's probably more clean to group all the view tests in a directory within the tests directory.